### PR TITLE
[WIP] Absorbing Understanding DVC into the User Guide

### PR DESCRIPTION
**Pull request by VANRao-Stack**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

This is no way the final version of what I aim to do, and some more editing and re-arranging is required, but I thought it would be a good idea for me to post this, so that we are all on the same page about the final structure of the user-guide. #425

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
> This is no way the final version of what I aim to do

For future ref. there's a couple better ways to express this: use a draft PR (see https://github.blog/2019-02-14-introducing-draft-pull-requests/), or at least add [WIP] to the title, which I just did 😉

File: content/docs/user-guide/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# User Guide
+# A More Comprehensive Understanding
 
-Our guides describe the main DVC concepts and features comprehensively,
+Our guides continues to describe the main DVC concepts and features comprehensively,
```
Comment (jorgeorpinel) [inline content/docs/user-guide/index.md:3]: Can you explain the reason of this change? I don't understand how it addresses the task at hand 🙁 

This file turns into the User Guide home at https://dvc.org/doc/user-guide.

**Comment by jorgeorpinel** [comment]
I'm confused by the path and naming of the files in this PR. Are you able to run the web app locally? It's all explained in the docs contrib guide, please go over that again.

Maybe the content changes here are usfeul so don't discard your branch, but I have to close this one too and ask you to go back to the whiteboard again please. See https://github.com/iterative/dvc.org/issues/425#issuecomment-633391398

Thanks!

**Comment by jorgeorpinel** [comment]
p.s. you can reach me or the team on Discord if you have questions about how the website works. We're happy to help

File: content/docs/user-guide/index.md (hunk 1 above)
Comment (VANRao-Stack) [reply content/docs/user-guide/index.md:3]: This wasn't supposed to be the user guide home, I'm sorry but I think one of the changes I Meade locally didn't show up here, I'll continue to work

**Pull request closed without merging**